de.variantsync.core/src/test/java/de/variantsync/core/ASTTest.java

```diff
@@ -0,0 +1,53 @@
+package de.variantsync.core;
+
+import static org.junit.Assert.assertEquals;
+import java.util.Arrays;
+
+import de.variantsync.core.ast.AST;
+import de.variantsync.core.ast.LineGrammar;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ASTTest {
+
+	AST<LineGrammar, String> ast;
+	AST<LineGrammar, String> mainJava;
+
+	@Before
+	public void setup() {
+		ast = new AST<>(LineGrammar.Directory, "src");
+		AST<LineGrammar, String> mainDir = new AST<>(LineGrammar.Directory, "main");
+		AST<LineGrammar, String> testDir = new AST<>(LineGrammar.Directory, "test");
+		mainJava = new AST<>(LineGrammar.TextFile, "Main.java");
+		AST<LineGrammar, String> emptyJava = new AST<>(LineGrammar.TextFile, "Empty.java");
+		AST<LineGrammar, String> emptyTetJava = new AST<>(LineGrammar.TextFile, "EmptyTest.java");
+		ast.addChild(testDir);
+		testDir.addChild(emptyTetJava);
+		ast.addChild(mainDir);
+		mainDir.addChild(mainJava);
+		mainDir.addChild(emptyJava);
+
+		mainJava.addChildren(
+				Arrays.asList(new AST<>(LineGrammar.Line, "public class Main {"), new AST<>(LineGrammar.Line, "    public static void main(String[] args)"),
```

Comment:
please use newlines for each entry here? Or is that prohibited by the styling conventions?

Reply:
Can we make the format not remove custom line breaks? But how should he differentiate between wanted and unwanted linebreaks?

Reply:
@h3ssto ?